astropy/io/votable/connect.py

```diff
@@ -179,3 +181,98 @@ def write_table_votable(
 io_registry.register_reader("votable", Table, read_table_votable)
 io_registry.register_writer("votable", Table, write_table_votable)
 io_registry.register_identifier("votable", Table, is_votable)
+
+
+# VOTable with embedded/linked Parquet file #
+def write_table_votable_parquet(tab, filename, column_metadata, overwrite):
+    """
+
+    This function allows writing a VOTable (XML) with PARQUET
+    serialization. This functionality is currently not
+    supported by Astropy (with the reason that this method
+    requires writing multiple files: a VOTable/XML and
+    PARQUET table). This function presents a wrapper, which
+    allows to do this. The concept is simple and probably
+    can be improved substantially. We first save the PARQUET
+    table using Astropy functionality. Then, we create a
+    VOTable with binary serialization. The latter is modified
+    later to include an external reference to the create
+    PARQUET table file.
+
+    Parameters
+    ----------
+    tab : astropy table
+        Contains the data
+    filename : str
+        The file name of the VOTable (e.g., "test.vot").
+        IMPORTANT: Needs to be a full path (i.e. no "../" or "./")
+    column_metadata : dictionary
+        Contains the metadata for the columns such as "unit" or
+        "ucd" or "utype".
+        (Example: column_metadata = {"id":{"unit":"","ucd":"meta.id","utype":"none"},
+        "mass":{"unit":"solMass","ucd":"phys.mass","utype":"none"} } )
```

Comment:
extreme nitpick: follow PEP8 in examples, too and format for readibility
```suggestion
        (Example: column_metadata = {"id": {"unit": "", "ucd": "meta.id", "utype": "none"},
                                     "mass": {"unit": "solMass", "ucd": "phys.mass", "utype": "none"} } )
```